_posts/2019-01-XX-enabling-cpb-on-pcengines-apu2.md

```diff
@@ -0,0 +1,416 @@
+---
+post_title: Enabling Core Performance Boost on PC Engines apu2
```

Comment:
Expanding the scope to AMD will not help at all IMO. Who does use AGESA in BIOS? How many open source firmware is there for AMD processors? I think the opposite actually - it may limit the scope, but if we just care about if someone clicks the link and enters the site, that's ok. But my kind wish is, that if someone enters, he/she will read the post.

I agree the EMV is significant and the headline may be improved to better attract the potential audience.